flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -420,31 +406,6 @@ static int32_t vtol_fsm_static()
  * @{
  */
 
-//! The currently configured navigation mode. Used to sanity check configuration.
-static enum vtol_nav_mode vtol_nav_mode;
-
-/**
- * General methods which based on the selected @ref vtol_nav_mode calls the appropriate
- * specific method
- */
-static int32_t do_default()
```

Comment:
ok i'll need some time to work through the implications of this change. this is where i wish we had unit tests on the whole nav system :(